src/vignette/util/consul.clj

```diff
@@ -0,0 +1,49 @@
+(set! *warn-on-reflection* true)
```

Comment:
ups my bad. This is leftover from early experiments with clojure.

On Thu, Aug 27, 2015 at 8:12 PM, Nelson Monterroso <notifications@github.com
> wrote:

> In src/vignette/util/consul.clj
> <https://github.com/Wikia/vignette/pull/95#discussion_r38127292>:
>
> > @@ -0,0 +1,49 @@
> > +(set! *warn-on-reflection* true)
>
> why is this needed? does it change it for the running service as a whole,
> or just for this namespace?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/vignette/pull/95/files#r38127292>.
>